src/views/deckviewer/alternate_art_popup_pane.ts

```diff
@@ -0,0 +1,108 @@
+import { BlobStorageDataLoader } from "../shared/client/blob_storage_data_loader";
+import { MiscOptions } from "../shared/client/cardfilters/filter_misc_options";
+import { FilterText } from "../shared/client/cardfilters/filter_text";
+import {
+  BaseCardRenderer,
+  CardRendererOptions,
+} from "../shared/client/renderers/base_card_renderer";
+import { CardRendererCompactDetails } from "../shared/client/renderers/card_renderer_compact_details";
+import { CardRendererCompactGrid } from "../shared/client/renderers/card_renderer_compact_grid";
+import { CardRendererCompactList } from "../shared/client/renderers/card_renderer_compact_list";
+import { CardRendererGrid } from "../shared/client/renderers/card_renderer_grid";
+import { CardRenderArea } from "../shared/client/renderers/card_render_area";
+import { DeckViewerViewBehavior } from "./behavior";
+
+export class AlternateArtPane {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private dl: BlobStorageDataLoader;
```

Comment:
This can use DataLoader instead of BlobStorageDataLoader. It doesn't require any of the things are are only in BlobStorageDataLoader

Reply:
No